egs/chime4/s5_2ch/run.sh

```diff
@@ -7,11 +7,22 @@
 #                Mitsubishi Electric Research Labs (Shinji Watanabe)
 #  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
 
+# Added advanced baseline
```

Comment:
this is not the place for a changelog: add your name above, e.g. after watanabe, say:
`#           2017   Szu-Jui Chen `
and remove this.